furms-core/src/test/java/io/imunity/furms/core/policy_documents/PolicyDocumentServiceImplTest.java

```diff
@@ -244,103 +239,6 @@ void shouldFindAllByCurrentUser() {
 		orderVerifier.verify(repository).findAllByUserId(eq(userId), any());
 	}
 
-	@Test
```

Comment:
why those tests were dropped? Do we have a replacement?

Reply:
Because method: findAllUsersWithoutCurrentRevisionPolicyAcceptance was removed